build_tools/benchmarks/upload_benchmarks_to_dashboard.py

```diff
@@ -82,16 +79,16 @@ def get_model_description(model_name: str, model_source: str) -> Optional[str]:
 
 def get_git_commit_hash(commit: str, verbose: bool = False) -> str:
   """Gets the commit hash for the given commit."""
-  return execute_cmd_and_get_output(['git', 'rev-parse', commit],
-                                    cwd=THIS_DIRECTORY,
-                                    verbose=verbose)
+  return benchmark_definition.execute_cmd_and_get_output(
+      ['git', 'rev-parse', commit], cwd=THIS_DIRECTORY, verbose=verbose)
 
 
 def get_git_total_commit_count(commit: str, verbose: bool = False) -> int:
   """Gets the total commit count in history ending with the given commit."""
-  count = execute_cmd_and_get_output(['git', 'rev-list', '--count', commit],
-                                     cwd=THIS_DIRECTORY,
-                                     verbose=verbose)
+  count = benchmark_definition.execute_cmd_and_get_output(
```

Comment:
Wait why is this command in "benchmark_definition"?

Reply:
I guess it's a historical issue? I can do some refactoring in a separate change.